FlowCrypt/Controllers/Compose/Extensions/ComposeViewController+MessageSend.swift

```diff
@@ -28,87 +28,27 @@ extension ComposeViewController {
         // https://github.com/FlowCrypt/flowcrypt-ios/issues/291
         try await Task.sleep(nanoseconds: 100 * 1_000_000) // 100ms
 
-        let sendableMsg = try await self.composeMessageService.validateAndProduceSendableMsg(
-            senderEmail: selectedFromEmail,
```

Comment:
Was `senderEmail` moved elsewhere?

Reply:
Yes, now we use `contextToSend.sender` as `senderEmail`